invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/MultipleOptionsSearchBar.js

```diff
@@ -32,7 +32,18 @@ export class MultipleOptionsSearchBar extends Component {
   render() {
     const { placeholder, options } = this.props;
     const { queryString } = this.state;
-
+    const button = (
+      <Button
+        icon
+        className="right-floated search"
+        onClick={() => {
+          const url = options[0]?.value;
```

Comment:
Does it make sense to have a `defaultOption` property so we can control what is default on empty search? You can always fallback on `options[0]`...